src/reconstruction/CoreModel.jl

```diff
@@ -375,56 +375,96 @@ function remove_reactions(m::CoreModel, rxns::Vector{String})
     end
 end
 
-"""
-    change_bounds!(
-        model::CoreModel,
-        rxns::Vector{Int};
-        xl::V = Float64[],
-        xu::V = Float64[],
-    ) where {V<:VecType}
+@doc @_change_bound_s_bang("CoreModel", "rxn_idxs", "Vector{Int}", "[2, 43]", true, true)
+function change_bounds!(
+    model::CoreModel,
+    reaction_idxs::Vector{Int};
+    lower_bounds = fill(-_constants.default_reaction_bound, length(reaction_idxs)),
+    upper_bounds = fill(constants.default_reaction_bound, length(reaction_idxs)),
+)   
+    for (rxn_idx, lb, ub) in zip(reaction_idxs, lower_bounds, upper_bounds)
+        change_bound!(model, rxn_idx; lower_bound=lb, upper_bound=ub)
+    end
+end
 
-Change the lower and/or upper bounds ('xl' and 'xu') for given reactions.
+@doc @_change_bound_s_bang("CoreModel", "rxn_idx", "Int", "2", false, true)
+function change_bound!(
+    model::CoreModel,
+    rxn::Int;
+    lower_bound = -_constants.default_reaction_bound,
+    upper_bound = _constants.default_reaction_bound,
```

Comment:
Hidden surprise:
```
change_bound!(m, 123, lower_bound=123)
```
... changes upper bound to 1000.

Reply:
yeah, this is definitely a hidden surprise... It complicates the code to fix it though

Reply:
so what about
```julia
function change_bounds(model, rxn; lower=missing, upper=missing)
  ismissing(lower) || ( m.rxn[i].lb = lower )
  ismissing(upper) || ( m.rxn[i].lb = upper )
end
```
btw you can remove `_bound` from parameter names, it's already there once.